src/index.js

```diff
@@ -1,35 +1,163 @@
+/* eslint-disable eqeqeq */
 import './style.css';
 
-const listContainer = document.querySelector('.list-container');
+const input = document.querySelector('.input-list');
+const submit = document.querySelector('.add');
+const tasksDiv = document.querySelector('.tasks');
+const deletebtn = document.querySelector('.deletebtn');
 
-const task1 = {
-  description: 'Fix my car',
-  completed: true,
-  index: 1,
-};
+// empty array
+let arrayOfTasks = [];
 
-const task2 = {
-  description: 'Watch series',
-  completed: false,
-  index: 2,
-};
+// ckeck if there is any thing in local storage
+if (localStorage.getItem('tasks')) {
+  arrayOfTasks = JSON.parse(localStorage.getItem('tasks'));
+}
+
+// trigger data from local storage
+// eslint-disable-next-line no-use-before-define
+getDataFromLocal();
 
-const task3 = {
-  description: 'Read articles',
-  completed: false,
-  index: 3,
+// submit tasks
+submit.onclick = () => {
+  // eslint-disable-next-line no-use-before-define
```

Comment:
- [ ] Please don't disable linter rules. These are here to write clean and standard code.